test/unit/math/opencl/indexing_rev_test.cpp

```diff
@@ -6,6 +6,7 @@
 #include <gtest/gtest.h>
 #include <algorithm>
 
+#ifndef STAN_TEST_SKIP_REQUIRING_OPENCL_INT64_BASE_ATOMIC
```

Comment:
Both tests are within the ifdef, so you can also move the includes in.

Reply:
Done.